src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/TierCalculationServiceTest.kt

```diff
@@ -143,75 +140,4 @@ internal class TierCalculationServiceTest {
       verify { tierCalculationRepository.findByCrnAndUuid(crn, calculationId) }
     }
   }
-
-  @Nested
-  @DisplayName("Calculate  Tier for Crn tests")
-  inner class CalculateTierForCrnTests {
-
-    @Test
-    fun `Should Call Collaborators Test value not changed`() {
-      every { assessmentApiService.getRecentAssessment(crn) } returns null // anything
-      every { assessmentApiService.getAssessmentNeeds(null) } returns mapOf() // anything
-      every { communityApiService.getDeliusAssessments(crn) } returns DeliusAssessments(BigDecimal.ZERO, 0) // anything
-      every { communityApiClient.getRegistrations(crn) } returns Pair(listOf(), listOf()) // anything
-      every { communityApiService.getConvictionsWithSentences(crn) } returns listOf() // anything
-
-      every { protectLevelCalculator.calculateProtectLevel(crn, any(), any(), any(), any()) } returns protectLevelResult
-      every { changeLevelCalculator.calculateChangeLevel(crn, any(), any(), any(), any(), any()) } returns changeLevelResult
-
-      every { tierCalculationRepository.findFirstByCrnOrderByCreatedDesc(crn) } returns validTierCalculationEntity
-
-      val slot = slot<TierCalculationEntity>()
-      every { tierCalculationRepository.save(capture(slot)) } answers { slot.captured }
-
-      service.calculateTierForCrn(crn)
-
-      // We don't update the SNS and recognise that is hasn't change.
```

Comment:
I think this is where the coverage has dropped.
I am not keen on seeing these tests come back though. Maybe there's another way using integration tests or BDD - causing two tier calculations perhaps?

Reply:
There is some BDD missing around "And an Update event is generated"